### DISCUSSION TITLE
Redesign blinds in blind mod

### DISCUSSION THREAD

**Discussion by alixdotsh** [ppy/osu#13111] [Ideas]
The blind mod in game seems unthemed, basically the way it was designed does not make sense. The mod itself is a good idea but the blinds should be designed, you have all these neon colors and all this all that in gameplay, but the blinds are just basic and seemed as if there was not really much attention put into it. If the blinds in game were redesigned to fit with maybe the default skin or with the default skin theme, it would look much nicer/cleaner in my opinion.

**Comment by peppy** [comment]
The blinds were made to look like shoji. They will probably be re-themed or skinnable in the future. Not sure this needs to stay open for now (might be quite distant).
